Enshrouded Server Manager/Form1.cs

```diff
@@ -413,29 +413,33 @@ private void DeleteProfile_Button_Click(object sender, EventArgs e)
                 "Error while deleting profile", MessageBoxButtons.OK, MessageBoxIcon.Error);
                     return;
                 }
-                // Delete the server folder
-                _fileSystemManager.Delete($"{Constants.Paths.SERVER_PATH}{selectedServerProfile}_delete");
 
-                // remove the profile
-                profiledata.Remove(serverProfile);
+                if (Directory.Exists($"{Constants.Paths.SERVER_PATH}{selectedServerProfile}_delete"))
+                {
+                    // Delete the server folder
+                    _fileSystemManager.Delete($"{Constants.Paths.SERVER_PATH}{selectedServerProfile}_delete");
 
-                // write the new profile to the json file
-                var output = JsonConvert.SerializeObject(profiledata, _jsonSerializerSettings);
-                File.WriteAllText($"{Constants.Paths.DEFAULT_PROFILES_PATH}{Constants.Paths.SERVER_PROFILES_JSON}", output);
+                    // remove the profile
+                    profiledata.Remove(serverProfile);
 
-                // Clear UI Elements
-                txtEditProfileName.Text = "";
-                lbxServerProfiles.Items.Remove(selectedServerProfile);
-                cbxProfileSelectionComboBox.Items.Remove(selectedServerProfile);
-                RefreshServerButtonsVisibility(selectedServerProfile);
+                    // write the new profile to the json file
+                    var output = JsonConvert.SerializeObject(profiledata, _jsonSerializerSettings);
+                    File.WriteAllText($"{Constants.Paths.DEFAULT_PROFILES_PATH}{Constants.Paths.SERVER_PROFILES_JSON}", output);
 
-                // reload form1
-                Form1_Load(sender, e);
+                    // Clear UI Elements
+                    txtEditProfileName.Text = "";
+                    lbxServerProfiles.Items.Remove(selectedServerProfile);
+                    cbxProfileSelectionComboBox.Items.Remove(selectedServerProfile);
+                    RefreshServerButtonsVisibility(selectedServerProfile);
```

Comment:
I don't believe we need these 3 lines if we call `Form1_Load({ender, e)`

Alternatively, we could leave these and *not* call `Form1_Load`